docs/adr/0010-raw-body-processor.md

```
@@ -0,0 +1,81 @@
+suggested a `strings.Builder` + `io.Copy` path for lower allocation pressure:
+
+> "`var buf strings.Builder; if _, err := io.Copy(&buf, reader); err != nil { … }`
+> — Generally can't be sure of eliding heap allocations but worth trying"
+> — @anuraaga ([review](https://github.com/corazawaf/coraza/pull/983#discussion_r1479118293))
+
+@jcchavezs pushed for `b.Len()` over a manual length tracker and unused
+parameter cleanup:
+
+> "Shall we use `b.Len` here? https://pkg.go.dev/strings#Builder.Len"
+> — @jcchavezs ([review](https://github.com/corazawaf/coraza/pull/983#discussion_r1478399917))
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)